docs/reference/api-conventions.asciidoc

```diff
@@ -378,6 +380,21 @@ supported units are:
 `tb`::  Terabytes
 `pb`::  Petabytes
 
+[[size-units]]
+[float]
+=== Data size units
```

Comment:
Call them unit-less quantities. Explain that it is things like "document count" and "queue depth", etc. It is unit-less in that it doesn't have a "unit" like "bytes" or "Hertz" or "meter" or "long tonne".